### ISSUE TITLE
Uninstall instructions

### ISSUE THREAD

**Issue by taoeffect** [mono/website#603]
Hi - there are installation instructions, but no uninstallation instructions. Seems like that's something important to include in the docs?

(Also, if anyone knows, how do you uninstall it on macOS?)

**Comment by taoeffect** [comment]
Oh, I found what seem to be the [uninstallation instructions](https://www.mono-project.com/docs/about-mono/supported-platforms/macos/#uninstalling-mono-on-macos) by searching the Github project for mono.

This is still an issue because:

1. Searching the web docs via the search bar for "uninstall" brings no results
2. The FAQ doesn't mention uninstallation
3. The documentation page for [installing on macOS](https://www.mono-project.com/docs/getting-started/install/mac/) doesn't mention how to uninstall either

In other words, it's very difficult to find the uninstallation instructions just via the docs website

**Comment by akoeplinger** [comment]
Thanks. There's actually a link to the uninstall instructions on the download page:
![image](https://user-images.githubusercontent.com/1376924/214873010-2ad36a07-5f3c-4090-9673-4ef7c979d435.png)

I've moved the script to the "Installing on macOS" page so hopefully it's also easier to find via search now.

**Comment by taoeffect** [comment]
Thank you!
